packages/updatePrescriptionStatus/tests/validation/testRequestContentValidation.test.ts

```diff
@@ -211,34 +211,80 @@ describe("Unit tests for validation of NHS number", () => {
 })
 
 describe("Unit tests for validation of status against business status", () => {
-  it.each([
-    {
-      taskStatus: "completed",
-      businessStatus: "With Pharmacy",
-      expected: "Completed state indicated for a prescription status requiring patient action."
-    },
-    {
-      taskStatus: "completed",
-      businessStatus: "Ready to collect",
-      expected: "Completed state indicated for a prescription status requiring patient action."
-    },
-    {
-      taskStatus: "completed",
-      businessStatus: "ReAdY tO cOlLeCt",
-      expected: "Completed state indicated for a prescription status requiring patient action."
-    },
-    {
-      taskStatus: "in-progress",
-      businessStatus: "With Pharmacy",
-      expected: undefined
-    },
-    {
-      taskStatus: "in-progress",
-      businessStatus: "Ready to collect",
-      expected: undefined
-    }
-  ])(
-    "When status is '$status' and business status is '$businessStatus', should return expected issue.",
+  const generateTestCase = (taskStatus: string, businessStatus: string, expected: string | undefined) => ({
```

Comment:
I think this test needs splitting, to make it clearer what the intention is.

One for each taskStatus, I'd say.

Reply:
The test was split based on taskStatus.